packages/fuel-indexer-tests/tests/e2e/graphql_api_postgres.rs

```diff
@@ -72,11 +72,14 @@ async fn test_can_return_query_response_with_all_fields_required_postgres() {
 
     server_handle.abort();
 
+    println!("status: {}", resp.status());
```

Comment:
No `println` in tests

Reply:
yep my bad, left in for my own debugging, will remove